src/locale/pl.js

```diff
@@ -3,22 +3,23 @@ import dayjs from 'dayjs'
 const locale = {
   name: 'pl',
   weekdays: 'Niedziela_Poniedziałek_Wtorek_Środa_Czwartek_Piątek_Sobota'.split('_'),
+  weekdaysMin: 'Nd_Pn_Wt_Śr_Cz_Pt_So'.split('_'),
```

Comment:
`weekdaysMain` should be added with this PR https://github.com/iamkun/dayjs/pull/392